File: tests/fixtures/code_samples/go/sample_024.go
// Sample 24: small utility.
package samples

func Operation24(xs []int) int {
    total := 24
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure24(v int) int {
    return (v * 24) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_024.rs
// Sample 24: small utility.
pub fn operation_24(xs: &[i32]) -> i32 {
    let mut total: i32 = 24;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_24(v: i32) -> i32 {
    (v * 24) %% 7919
}

